src/src/PluginStructs/P093_data_struct.h

```diff
@@ -118,12 +118,13 @@ struct P093_data_struct : public PluginTaskData_base {
     ReadTimeout
   };
 
-  static const uint8_t Temperature = 0x01;
-  static const uint8_t Power       = 0x02;
-  static const uint8_t Mode        = 0x04;
-  static const uint8_t Fan         = 0x08;
-  static const uint8_t Vane        = 0x10;
-  static const uint8_t WideVane    = 0x20;
+  static const uint8_t Temperature        = 0x01;
+  static const uint8_t Power              = 0x02;
+  static const uint8_t Mode               = 0x04;
+  static const uint8_t Fan                = 0x08;
+  static const uint8_t Vane               = 0x10;
+  static const uint8_t WideVane           = 0x20;
+  static const uint8_t RemoteTemperature  = 0x07;
```

Comment:
@jfmennedy these flags are being used as a bitmap, not as a numeric value. If you change this to `0x30` the temperature, power and mode commands should start to work as intended again 😅

Reply:
will try this now..  saw this comment too late before my last commit and comment...

Reply:
That´s it :-) It is working :-) Thank you sooooo much :-)